recipes/wagtail-markdown

```diff
@@ -0,0 +1,45 @@
+{% set name = "wagtail-markdown" %}
+{% set version = "0.4.1" %}
+{% set sha256 = "89b24b04d874feb97adb0299b5fbc06ea29b5a835ef765b82846575a1d77f63f" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/torchbox/wagtail-markdown/archive/0.4.1.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - Markdown ==2.6.2
+    - bleach ==1.4.2
+
+test:
+  imports:
+    - wagtailmarkdown
+
+about:
+  home: https://github.com/torchbox/wagtail-markdown
+  license: zlib
+  license_family: zlib
+  license_file: LICENSE
```

Comment:
Apparently there is no license file. Could you please raise an issue upstream and ask them to include the license file?

Reply:
Done